crates/bitwarden/src/tool/exporters/client_exporter.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{
+    error::Result,
+    tool::exporters::{export_organization_vault, export_vault, ExportFormat},
+    vault::{CipherView, CollectionView, FolderView},
+    Client,
+};
+
+pub struct ClientExporters<'a> {
+    pub(crate) _client: &'a crate::Client,
+}
+
+impl<'a> ClientExporters<'a> {
+    pub async fn export_vault(
+        &self,
+        folders: Vec<FolderView>,
```

Comment:
I'm assuming when using the export, the application won't have the list of folders and ciphers already decrypted, at most it will have the list view models. Would it make more sense to ask for the encrypted version and do it all in a single FFI call?

Reply:
Yes, I had that on my mind but forgot it during the weekend.